CI: use images from gnuradio docker repository

wirew0rm opened the pull request:
@mormj moved the docker images from his personal repo into the gnuradio-docker repository in a gr-4.0 branch:
https://github.com/gnuradio/gnuradio-docker/tree/dev-4.0 This commit uptdates the CI to use the images from there so now everything lives inside the gnuradio organisation now.

mormj commented:
I just updated the container names as ghcr.io/gnuradio/gnuradio-docker:fedora-36-4.0 etc. so as to not have conflicting names

wirew0rm commented:
> I just updated the container names as ghcr.io/gnuradio/gnuradio-docker:fedora-36-4.0 etc. so as to not have conflicting names

updated to the new image names :heavy_check_mark:
